packages/elements-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-core",
-  "version": "8.1.0",
+  "version": "8.1.1",
```

Comment:
The change happens only in `dev-portal` so we don't need to bump `elements-core`